src/analysis/complex/liouville.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2022 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+-/
+import analysis.complex.cauchy_integral
+import analysis.calculus.fderiv_analytic
+
+/-!
+# Liouville's theorem
+
+In this file we prove Liouville's theorem: if `f : E → F` is complex differentiable on the whole
+space and its range is bounded, then the function is a constant. Various versions of this theorem
+are formalized in `differentiable.apply_eq_apply_of_bounded`,
+`differentiable.exists_const_forall_eq_of_bounded`, and
+`differentiable.exists_eq_const_of_bounded`.
+
+The proof is based on the Cauchy integral formula for the derivative of an analytic function, see
+`complex.deriv_eq_smul_circle_integral`.
+-/
+
+open topological_space metric set filter asymptotics function measure_theory
+open_locale topological_space filter nnreal real
+
+universes u v
+variables {E : Type u} [normed_group E] [normed_space ℂ E]
+  {F : Type v} [normed_group F] [normed_space ℂ F] [measurable_space F] [borel_space F]
+    [second_countable_topology F] [complete_space F]
+
+namespace complex
+
+/-- If `f` is complex differentiable on a closed disc with center `c` and radius `R > 0`, then
+`f' c` can be represented as an integral over the corresponding circle.
+
+TODO: add a version for `w ∈ metric.ball c R`.
+
+TODO: add a version for higher derivatives. -/
+lemma deriv_eq_smul_circle_integral {R : ℝ} {c : ℂ} {f : ℂ → F} (hR : 0 < R)
+  (hc : continuous_on f (closed_ball c R)) (hd : differentiable_on ℂ f (ball c R)) :
+  deriv f c = (2 * π * I : ℂ)⁻¹ • ∮ z in C(c, R), (z - c) ^ (-2 : ℤ) • f z :=
+begin
+  lift R to ℝ≥0 using hR.le,
+  refine (has_fpower_series_on_ball_of_continuous_on_of_differentiable_on
+    hc hd hR).has_fpower_series_at.deriv.trans _,
+  simp only [cauchy_power_series_apply, one_div, zpow_neg₀, pow_one, smul_smul,
+    zpow_two, mul_inv₀]
+end
+
+/-- If `f` is continuous on a closed disc of radius `R`, is complex differentiable on its interior,
+and its values on the boundary circle of this disc are bounded from above by `C`, then the norm of
+its derivative at the center is at most `C / R`.
+
+TODO: drop unneeded assumptions `[complete_space F] [second_countable_topology F]`.  -/
+lemma norm_deriv_le_of_forall_mem_sphere_norm_le {c : ℂ} {R C : ℝ} {f : ℂ → F} (hR : 0 < R)
```

Comment:
can you already take out the assumptions `borel_space F` and `measurable_space F` out of this lemma and the next ones, by `letI`-ing them in the proof?

Reply:
Done.